vite/vite.config.prod.ts

```diff
@@ -0,0 +1,10 @@
+import { mergeConfig, type UserConfig } from 'vite';
+
+import baseOptions from './vite.config.base';
+
+export default mergeConfig<UserConfig, UserConfig>(baseOptions, {
+  build: {
+    cssMinify: true,
+    target: ['es2020', 'chrome95', 'edge95', 'firefox94', 'safari14'],
```

Comment:
cactbot only supports Safari 16.4 or higher (due to lookbehind in regular expressions, see <https://caniuse.com/js-regexp-lookbehind>). Not sure if it'll impact the build at all, but figured I'd point that out.

Reply:
Oh, you're right, I missed that one.